internal/auth/authenticator_test.go

```diff
@@ -424,7 +418,7 @@ func TestSignIn(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			config := testConfiguration(t)
 			auth, err := NewAuthenticator(config,
-				setMockValidator(tc.validEmail),
+				SetValidators([]options.Validator{options.NewMockValidator(tc.validEmail)}),
```

Comment:
Would we prefer to stick with a custom 'setValidator' function or similar to use within these tests?

Reply:
This seems fine as written.